docs/generated/sql/bnf/alter_role_stmt.bnf

```diff
@@ -1,15 +1,61 @@
+	| 'ALTER' 'ROLE' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET' session_var
+	| 'ALTER' 'ROLE' string_or_placeholder  'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' string_or_placeholder  'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' string_or_placeholder  'RESET_ALL' 'ALL'
+	| 'ALTER' 'ROLE' string_or_placeholder  'RESET' session_var
+	| 'ALTER' 'USER' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET_ALL' 'ALL'
+	| 'ALTER' 'USER' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET' session_var
+	| 'ALTER' 'USER' string_or_placeholder  'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' string_or_placeholder  'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' string_or_placeholder  'RESET_ALL' 'ALL'
+	| 'ALTER' 'USER' string_or_placeholder  'RESET' session_var
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET_ALL' 'ALL'
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET' session_var
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder  'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder  'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder  'RESET_ALL' 'ALL'
+	| 'ALTER' 'ROLE' 'IF' 'EXISTS' string_or_placeholder  'RESET' session_var
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET_ALL' 'ALL'
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder 'IN' 'DATABASE' database_name 'RESET' session_var
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder  'SET' var_name '=' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder  'SET' var_name 'TO' var_value ( ( ',' var_value ) )*
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder  'RESET_ALL' 'ALL'
+	| 'ALTER' 'USER' 'IF' 'EXISTS' string_or_placeholder  'RESET' session_var
+	| 'ALTER' 'ROLE_ALL' 'ALL' 'IN' 'DATABASE' database_name 'SET' var_name '=' var_value ( ( ',' var_value ) )*
```

Comment:
> is this how it will show up in the docs page?

Yes.

> in reality, the command would be ALTER ROLE ALL ... (and similar for ALTER USER ALL ...). do we need a replacement rule for that?

Probably. Just added one.